exporter/datadogexporter/internal/metrics/sketches.go

```diff
@@ -15,7 +15,7 @@
 // Parts of this package are based on the code from the datadog-agent,
 // https://github.com/DataDog/datadog-agent/blob/main/pkg/metrics/sketch_series.go
 
-package sketches // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/datadogexporter/internal/sketches"
```

Comment:
I would like to keep the sketches part in a separate package: this code is vendored from the Datadog Agent codebase and I want to keep it separate from the rest so we can at some point import it from the Agent. Could you move this to `internal/metrics/sketches` instead?

Reply:
I did exactly that. I also added a package-level comment with a `TODO(mx-psi)`. I hope that's ok.